index.html

```diff
@@ -4804,6 +5031,57 @@ <h4><dfn>Get Active Element</dfn></h4>
   <p>Otherwise, return <a>error</a> with <a>error code</a> <a>no such element</a>.
 </ol>
 </section> <!-- /Get Active Element -->
+
+<section>
+<h4><dfn>Get Element Shadow Root</dfn></h4>
+
+<table class="simple jsoncommand">
+  <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+  </tr>
+  <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/element/{<var>element id</var>}/shadow</td>
+  </tr>
+</table>
+
+<p>The <a>remote end steps</a> are:
+
+<ol>
+  <li><p>If the <a>current browsing context</a> is <a>no longer
+  open</a>, return <a>error</a> with <a>error code</a> <a>no such
+  window</a>.
+
+  <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
+
+  <li><p>Let <var>element</var> be the result
+    of <a>trying</a> to <a>get a known connected element</a>
+    with <a>url variable</a> <var>element id</var>.  
+
+  <li><p>Let <var>shadow root</var> be the result of getting an
+    <a>own property</a> named "shadowRoot" from <var>element</var>.
+
+  <li><p>If <var>shadow root</var> is undefined, return <a>error</a>
+    with <a>error code</a> <a>no such shadow root</a>.
+
+  <li><p>Let <var>shadow root node type</var> be the result of
+    getting an <a>own property</a> named "nodeType" from
+    <var>shadow root</var>.
+
+  <li><p>If <var>shadow root node type</var> is undefined, or has
+    a value other than 11, return <a>error</a> with <a>error code</a>
```

Comment:
Does that `11` equate to a constant in a spec somewhere?

Reply:
Will add reference to DOCUMENT_FRAGMENT_NODE from the DOM spec.